lib/fluent/plugin/buffer.rb

```diff
@@ -372,7 +372,7 @@ def write(metadata_and_data, format: nil, size: nil, enqueue: false)
       end
 
       def queue_full?
-        @queued_chunks_limit_size && (synchronize { @queue.size } >= @queued_chunks_limit_size)
+        synchronize { @queue.size } >= @queued_chunks_limit_size
```

Comment:
buffer test complains `@queued_chunks_limit_sise` is nil and not to be able to compare.
Could you check it?
https://travis-ci.org/fluent/fluentd/jobs/452714881#L547